src/utils/data-formatters/schema/pending-history-event-schema.ts

```diff
@@ -0,0 +1,59 @@
+import { z } from 'zod';
+
+import { PendingActivityState } from '@/__generated__/proto-ts/uber/cadence/api/v1/PendingActivityState';
+import { PendingDecisionState } from '@/__generated__/proto-ts/uber/cadence/api/v1/PendingDecisionState';
+
+const timestampSchema = z.object({
```

Comment:
Thoughts on having these schemas in a shared location? This isn't necessary now, but I think we can discuss

Reply:
We don't need to do it right away, we can look at it later as well